cmd/wallet-web/src/pages/chapi/wallet/register/register.js

```diff
@@ -59,9 +59,15 @@ export class RegisterWallet extends WalletManager {
             console.warn("unable to find to mediator wallet URL, registered wallet may not support DID Exchange")
         }
 
+        console.info('time taken to register mediator : ' + (new Date().getTime() - start))
+        var startTime = new Date().getTime();
+
         // create DID
         let did = await this.didManager.createTrustBlocDID(keyType, signType)
 
+        console.info('time taken to create trustbloc did : ' + (new Date().getTime() - startTime))
```

Comment:
@rolsonquadras @fqutishat  If creating trustbloc did becomes performance bottleneck we could create it asynchronously (at the beginning of register flow)